libraries/Microsoft.Bot.Builder.Core.Extensions/Microsoft.Bot.Builder.Core.Extensions.csproj

```diff
@@ -58,6 +54,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.CSharp" Version="4.4.1" />
     <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
+    <PackageReference Include="System.Dynamic.Runtime" Version="4.3.0" />
```

Comment:
I would *STRONGLY* like to avoid pulling in system.dynamic to the Extensions package. 

How can we avoid doing this?

Reply:
Hmmm.  I removed it and it seems to work fine--even though there are some dynamics in here.